cpp/oneapi/dal/algo/linear_regression/test/batch.cpp

```diff
@@ -34,10 +34,10 @@ class lr_batch_test : public lr_test<TestType, lr_batch_test<TestType>> {
     using train_result_t = typename base_t::train_result_t;
 
     void generate_dimensions() {
-        this->t_count_ = GENERATE(327, 888, 32777);
-        this->s_count_ = GENERATE(111, 333, 16777);
-        this->f_count_ = GENERATE(2, 17, 99);
-        this->r_count_ = GENERATE(2, 15, 33);
+        this->t_count_ = GENERATE(307, 12999, 17777);
+        this->s_count_ = GENERATE(111, 777);
+        this->f_count_ = GENERATE(2, 17);
+        this->r_count_ = GENERATE(2, 15);
```

Comment:
Reducing dims?

Reply:
Yeah. It takes just too much time to evaluate all variants especially on relatively weak hardware